sections/5_reproducibility/reproducibility_composite_confidence_index.qmd

```diff
@@ -0,0 +1,249 @@
+
+- **OpenAIRE Research Graph**  
+  [OpenAIRE](https://graph.openaire.eu/) aggregates metadata on publications, datasets, and software. It supports linking artefacts to publications and can be used to identify reuse cases and citances that indicate how artefacts are cited, which is essential for FWRI and RCI.  
+
+- **OpenAlex**  
+  [OpenAlex](https://openalex.org/) is an openly accessible bibliometric database that provides citation counts, references, and links to associated datasets and software. It can be used to calculate citation-based metrics such as FWCI and to identify citation links needed for FWRI and RCI.  
+
+- **Dimensions**  
+  [Dimensions](https://app.dimensions.ai/) offers citation data and normalised indicators such as the Field Citation Ratio (FCR). It provides expected citation baselines by field and year, which are useful for calculating FWCI.  
+
+- **Scopus**  
+  [Scopus](https://www.scopus.com/) is a large citation database that includes the Field-Weighted Citation Impact (FWCI) indicator. It can serve as a source for citation data and normalised impact values used in RCCI.  
+
+- **Web of Science / InCites**  
+  [Web of Science](https://webofscience.com/) provides citation data and normalised citation metrics through InCites, where the Category Normalised Citation Impact (CNCI) is implemented. This can be used as an alternative to Scopus FWCI or Dimensions FCR.  
+
+- **DataCite**  
+  [DataCite](https://datacite.org/) is a registry that provides persistent identifiers (DOIs) and metadata for research datasets and software. It is especially useful for retrieving metadata elements (Name, Version, License, URL) needed for calculating the FAIR Index.  
+
+- **Crossref**  
+  [Crossref](https://www.crossref.org/) maintains extensive metadata for scholarly publications and related outputs, including references and links to datasets and software. It is valuable both for reuse tracking (FWRI) and FAIR metadata extraction (FI).  
+
+- **Zenodo / Figshare / Institutional Repositories**  
+  These repositories host datasets, software, and other artefacts. They expose metadata via APIs, which can be used to evaluate FAIRness and retrieve usage information for reuse analysis.  
+
+- **scite.ai**  
+  [scite.ai](https://scite.ai/) provides classification of citation statements into supporting, refuting, or mentioning. It can be used to measure polarity of publications and calculate the RCI.  
+
+---
+
+# Existing Methodologies
+
+## SciNoBo Toolkit
+
+The [SciNoBo Toolkit](https://scinobo.ilsp.gr/toolkit) has implemented and operationalised the RCCI and its component indicators into a **working monitoring dashboard**.  
+
+- In the **TIER2 project**, SciNoBo was used to extract artefacts from project deliverables and publications, link them to citation and reuse data, and compute FWCI, FWRI, FI, RCI, and RCCI.  
```

Comment:
Could you provide a URL for TIER2?

Reply:
yes, I'll add https://tier2-project.eu/ here